small/matras.c

```diff
@@ -364,6 +364,34 @@ matras_touch_reserve(struct matras *m, int count)
 	return matras_allocator_reserve(m->allocator, max_extents_required);
 }
 
+#define MATRAS_DIV_ROUND_UP(a, b) ((a) + (b) - 1) / (b)
+
+/**
+ * Reserve the max amount of extents required to successfully allocate @p count
+ * blocks. The extents are reserved in the allocator given on construction.
+ */
+int
+matras_alloc_reserve(struct matras *m, int count)
```

Comment:
Please add a unit test to cover this code.

Reply:
This part of the patch had been required for RTREE update, but it's been [deferred](https://github.com/tarantool/tarantool/issues/10831), so the changes are dropped.